### DISCUSSION TITLE
Remove "site name" from every page title?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [squidfunk/mkdocs-material#5265] [Q&A] [answered]
All pages in my docs ends with `- <my site name>`. Is there anyway to prevent pages to end with this?

The reason for why I would like to do that is because, when using Giscus I would like to use the page title to create a discussion. It looks really weird when `- <my site name>` is always appended to the title.

**Comment by squidfunk** [comment, marked as answer]
Sure, you can override the title using [theme extension](https://squidfunk.github.io/mkdocs-material/customization/#overriding-blocks), specifically the `htmltitle` block:

https://github.com/squidfunk/mkdocs-material/blob/6a4be6b723b35388a388b8946ff898db0fcc9b5d/src/base.html#L74-L82

Something like this:

``` jinja2
{% extends "base.html" %}

{% block htmltitle %}
  {% if page.meta and page.meta.title %}
    <title>{{ page.meta.title }}</title>
  {% elif page.title and not page.is_homepage %}
    <title>{{ page.title | striptags }}</title>
  {% else %}
    <title>{{ config.site_name }}</title>
  {% endif %}
{% endblock %}
```

**Reply by fredrikaverpil** [reply]
Awesome, thanks! 😄
